web/book/src/queries/variables.md

```diff
@@ -71,4 +107,4 @@ table recursive_example = (s"""
 
 from recursive_example
 
--->
+```
```

Comment:
Now this could be implemented using `loop`, but this is not the place for it.

This example can be removed.